resolwe_bio/tools/pathway_predictor.py

```diff
@@ -1,7 +1,26 @@
 #!/usr/bin/env python3
```

Comment:
It much better to save all the output to a json file and than call the ``re-save pathways my_file.json`` in the process itself. For each addition to RESULTS list, you could print out something like ``print('{"proc.progress":0.3}'``. The progress value will show in the bioinfo dashboard as an analysis progress (30 % done in that case).

Reply:
but in this case the json file would be saved to MongoDB only after the script finished running, and what is needed is that the entry in MongoDB is updated with new set of reactions during the script execution